code/src/js/Form.js

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from 'react'
+
+import '../css/survey.css'
+import '../css/button.css'
+
+import { AgeQuestion, DrinkQuestion, AlcoholQuestion } from './Questions.js'
+import { Result } from './Result.js'
+
+export const Form = () => {
+
+  const [legal, setLegal] = useState(false)
+  const handleLegalChange = newLegal => {
+    setLegal(newLegal)
+  }
+
+  const [drink, setDrink] = useState("aperol")
+  const handleDrinkChange = newDrink => {
+    setDrink(newDrink)
+  }
+
+  const [alcohol, setAlcohol] = useState("Conventional")
+  const handleAlcoholChange = newAlcohol => {
+    setAlcohol(newAlcohol)
+  }
+
+  const [result, setResult] = useState(false)
+  const handleSubmit = event => {
+    event.preventDefault()
+    setResult(true)
+  }
```

Comment:
This is so nice! You're very good at naming your variebles! Makes it easier to understand 👍